### PR TITLE
Updating csv modeling and readme

### PR DESCRIPTION
**Pull request by corps**
Followup on model migration.

1. magma development no longer requires checking out the magma repositories
2. removed project_names from development config
3. Updating readme

initialize projects and test labors setup is handled using fixture files.  There is some open work around supporting `column_name` and `dictionary` in the CSV that would further simplify, but that is left for a future PR.

### CODE REVIEW

File: etna/lib/etna/clients/magma/workflows/json_validators.rb [hunk 1]
```diff
@@ -172,10 +172,6 @@ def validate_links
           link_attributes.each do |attribute|
             check_key("attribute #{attribute.attribute_name}", attribute.raw, 'link_model_name')
 
-            if attribute.attribute_name != attribute.link_model_name
```
Comment (corps) [inline etna/lib/etna/clients/magma/workflows/json_validators.rb:175]: This _used_ to be an issue, but it is no longer.

File: etna/lib/etna/clients/magma/workflows/model_synchronization_workflow.rb [hunk 2]
```diff
@@ -161,7 +161,7 @@ def ensure_model_link(model_name, link_model_name, attribute_name)
           return if target_attributes.attribute_keys.include?(target_link_model_name)
 
           add_link = AddLinkAction.new
-          add_link.links << AddLinkDefinition.new(model_name: target_model_name, attribute_name: target_link_model_name, type: source_attribute.attribute_type)
+          add_link.links << AddLinkDefinition.new(model_name: target_model_name, attribute_name: attribute_name, type: source_attribute.attribute_type)
```
Comment (corps) [inline etna/lib/etna/clients/magma/workflows/model_synchronization_workflow.rb:164]: This bug duplicated link attributes into two attributes: the intended one, and one named after the link_model.  This is intended behavior captured correctly.

**Comment by coleshaw** [comment]
nit: probably not strictly necessary, but is it also possible then to clean up [initialize-projects.sh](https://github.com/mountetna/monoetna/blob/master/magma/development/initialize-projects#L23)?

**Review by coleshaw — APPROVED** [review]

**Comment by corps** [comment]
> nit: probably not strictly necessary, but is it also possible then to clean up [initialize-projects.sh](https://github.com/mountetna/monoetna/blob/master/magma/development/initialize-projects#L23)?

To do that, we'd still need a hook for test to setup the test project.  Yeah, that's kind of the last pin in removing all this code, hmm.

**Pull request merged**